### PR TITLE
Dark Theme Improvements

### PR DESCRIPTION
**Pull request by sgraband**
Dark theme is now supported by the diagram editor and the InputField for renaming Attributes/Classes is now better readable in both Themes.
Light Theme, does not have a white background anymore, due to the way the theme colors in theia work. A follow-up issue will be submitted (#70).
Inheritance edges now have the same color as other edges.

New Light Theme:
LightTheme

New Dark Theme:
DarkTheme

Renaming looks the following in Dark Theme:
Renaming2

Resolves Issue #51

### CODE REVIEW

File: client/sprotty-ecore/css/diagram.css [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
     width: 100%;
     height: 500px;
     border-style: solid;
     border-width: 1px;
-    border-color: #bbb;
+    border-color: var(--sprotty-border);
+}
+
+.label-edit input {
+    background: rgba(255, 255, 255, 0.5);
```
Comment (eneufeld) [inline client/sprotty-ecore/css/diagram.css:17]: doesn't this look weird in dark theme? I didn't test it, but color wise this looks weird.

File: client/sprotty-ecore/css/diagram.css [hunk 2]
```diff
@@ -146,23 +165,19 @@ svg {
 }
 
 .ecore-edge>.triangle.inheritance {
-    fill: white;
-}
-
-.ecore-edge.inheritance {
```
Comment (eneufeld) [inline client/sprotty-ecore/css/diagram.css:152]: why is this removed completely?

**Comment by eneufeld** [comment]
@simongraband I like the use of variables. 
Could you attach screenshots of your fix with light and dark theme?

File: client/sprotty-ecore/css/diagram.css (hunk 1 above)
Comment (sgraband) [reply client/sprotty-ecore/css/diagram.css:17]: I attached a screenshot in the description. I think it looks fine.

File: client/sprotty-ecore/css/diagram.css (hunk 2 above)
Comment (sgraband) [reply client/sprotty-ecore/css/diagram.css:152]: I talked to @JonasHelming and he thought, that the inheritance edges should have the same color as the other edges. And since the only purpose of the ecore-edge.inheritance attribute was changing the color i removed it.

**Comment by sgraband** [comment]
> @simongraband I like the use of variables.
> Could you attach screenshots of your fix with light and dark theme?

Are the ones i provided enough? Or should i upload a few more?

**Review by eneufeld — DISMISSED** [review]
Looks good to me

**Comment by eneufeld** [comment]
@simongraband could you rebase so that we don't have a conflict?

**Review by eneufeld — APPROVED** [review]
